docs/ribs.docs

```diff
@@ -99,6 +99,51 @@ and given that `time` was the "time since the program started", this
 would draw a sprite at (100, 100) shrunk to half size along the y axis, rotating
 around it's center.
 
+# `draw_text(text, position, size=32, color=pg.Color(255, 255, 255), font=None)`
+Draw `text` at `position`, which is given in pixels from the top left corner.
+Optional arguments include `size` given in pixels, `color` which is
+a standard pygame color and `font` which is a string. To get a list of
+available fonts, use `pg.font.get_fonts()`.
+!--params
+[text] The text to be drawn.
+[position] The top left corner of the text, in pixel coordinates.
+[size] The font size in pixels. (Optional)
+[color] The color of the text. (Optional)
+[font] Name of the font to use. (Optional)
+--!
+
+## ex
+!--code
+draw_text("Level: 1", (0, 0))
+--!
+Draw "Level: 1" in the top left corner of the screen.
+
+## ex
+!--code
+draw_text("Level: 2", (0, 0), size=100)
+--!
+Draw "Level: 2" with large letters in the top left corner of the screen.
+
+## ex
+!--code
+draw_text("Level: 3", (0, 0), color=pg.Color(255, 0, 0))
+--!
+Draw "Level: 3" in red in the top left corner of the screen.
+
+## ex
+!--code
+draw_text("Level: 4", (0, 0), font="Fira Code")
```

Comment:
Can we use a more common font here? I imagine a few people may copy paste this code and see no effect

Reply:
Good idea, do you know any common fonts? It can vary drastically depending on operating system. Maybe just some windows font?